scripts/provisioning/upgradecmd.py

```diff
@@ -38,6 +39,23 @@ def process(self):
     self.logger.info(f"Processing {self.name} {self.url}")
     self.logger.info("validations started")
     self.phase_prereqs_validate(self.name)
-    self.phase_keys_validate(self.url, self.name)
-    self.validate_config_files(self.name)
     self.logger.info("validations completed")
+
+    # merge_configs() is imported from the merge.py
+    # Upgrade config files
+    self.logger.info("merge configs started")
+    merge_configs()
+    self.logger.info("merge configs completed")
+
+    # Remove temporary .old files from /tmp
+    self.logger.info("Remove sample.old files started")
+    path = "/tmp"
```

Comment:
check if any old sample file is present in /tmp directory or not

Reply:
DeleteFileOrDirWithRegex() will handle this case 
if file is not present then it will skip the removal and will check for next file